Gemfile

```diff
@@ -70,17 +70,14 @@ end
 group :development, :test do
   gem 'pry' # pop `pry` in controller code to open up an IRB terminal
   gem 'byebug' # pop `byebug` in view code for open up an IRB terminal
-  gem 'knapsack' # lets us split up our tets so they run faster in CI
```

Comment:
We don't need this anymore due to parallelizing on one machine rather than across machines, like circle does.